### ISSUE TITLE
Remove ability to edit image flag

### ISSUE THREAD

**Issue by Sherlock21** [FreeUKGen/FreeUKRegProductIssues#824]

**Comment by Sherlock21** [comment]
A batch  (vicryall.SOMILMBU5.csv) appears to have been uploaded with image numbers and the file header shows +LDS correctly.
One record viewed online shows an entry in the Image Number fields BUT the counter in the summary says : There are NO images. - which appears not to be correct based on the  online view of record number 1 in the batch. 

YET when  I downloaded the file concerned, there are NO image numbers in cols O & P.

What's gone wrong please? it appears to be a bug.

This issue is entered  on behalf of the CC for SOM.

**Comment by Captainkirkdawson** [comment]
Points to be considered.
1. The original file had the +LDS flag set and image numbers on records.
2. The file was replaced on Oct 1 and the replacing file had been downloaded and the new replacing file did not contain any image numbers
3. It is possible for the transcriber or coordinator to have edited the header  and changed the image flag to no. If so on the download the image numbers would have been eliminated.

My conclusion is that the file had been correctly uploaded. The transcriber/coordinator had edited the image field to say no . Then downloaded the file done some changes and replaced the original file but with one that had no image numbers

No bug detected.

But should a transcriber/coordinator be allowed to delete the image flag?

**Comment by Sherlock21** [comment]
I can't help wondering Why it is possible to change the flag: Image Numbers Yes/ No.
Surely that is something that would be derived from the presence or absence of the +LDS marker rather than whether there is an actual number on the data row ( since not all rows may have an image number).

As it currently configured, one can change the Yes/NO on Edit Batch Header screen and it does nothing to the image numbers in the data.

Indeed: why does the Batch Header screen need to show why there are image numbers in the data when one can just look at the data online.

I am of the view that if the flag: Image Numbers YES/NO is to remain in the Batch Header details, so it can be utilised when configuring a file to Download, then it should be set:-
 by the uploaded original file,
 and revised by every Reloaded file if appropriate.
-ie if its to have any serious value, then it must be maintained accurate.

**Comment by Captainkirkdawson** [comment]
Certainly a spec change is possible. Views of others is likely appropriate.
There was a reason for it to be edited but cannot recall the circumstances

**Comment by Sherlock21** [comment]
I have looked at the files in theAttic for this one.
the oldest has image numbers.
the next oldest does not have. (i.e. the one with date put into Attic of
 2016-10-19T14:08:02+00:00.

The Images flag currently says NO, so can you see from the audit trail if it was changed near that timestamp? or before, or after then. - as we can't  get response from the CC concerned.
Why do I ask: its a strange idea that the CC would go change this Images marker. so I am thinking/ wondering did it come in with the first Reloaded file?

**Comment by Captainkirkdawson** [comment]
I took the first attic file (which was replaced on October 1) and loaded into my system and it loads totally correctly, images flag is set and the image numbers are there.. For your information the other ones if loaded also set the images flag but there are no image numbers in the files so nothing there. 
If you attempt to download a batch that has the images flag set to no images then it does not try to write the image fields. This is done so that users downloading batches do not suddenly see a bunch of empty fields in their records.
I have not made any attempt to locate one user action in the log files.
